### PR TITLE
Cephadm: Add NFS support

### PR DESCRIPTION
**Pull request by sunilkumarn417**
Signed-off-by: sunilkumarn417 <sunnagar@redhat.com>

- Added NFS support.

**Test results**
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1614095651953/result.html

### CODE REVIEW

**Review by psathyan — APPROVED** [review]
At some point of time, we may want to think about restricting the scope of `cephadm.yaml` test suite and create a new one. Otherwise we might end up having a big test suite and have difficulty maintaining it.

**Comment by sunilkumarn417** [comment]
> At some point of time, we may want to think about restricting the scope of `cephadm.yaml` test suite and create a new one. Otherwise we might end up having a big test suite and have difficulty maintaining it.
I Strongly agree on this and we should address which tests to be part of sanity and feature test-suite.

File: tests/cephadm/test_nfs.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    """
+    Ceph-admin module to manage NFS service
+
+    Args:
+        ceph_cluster (ceph.ceph.Ceph): Ceph cluster object
+        kw: test data
+
+    check ceph.ceph_admin.nfs for test config
+    """
+    log.info("Running Ceph-admin Rados-GW( RGW ) test")
```
Comment (ktdreyer) [inline tests/cephadm/test_nfs.py:19]: This is a mistaken copy-and-paste from `tests/cephadm/test_rgw.py`, mind updating this?

Comment (sunilkumarn417) [reply tests/cephadm/test_nfs.py:19]: Addressed in https://github.com/red-hat-storage/cephci/pull/385

**Pull request merged**